### ISSUE TITLE
Update PDF Generator standard template

### ISSUE THREAD

**Issue by ArakTaiRoth** [GovAlta/ui-components#1315]
# Intro
There's a project run by ADSP, [PDF Generator](https://govalta.github.io/adsp-monorepo/services/pdf-service.html). It uses templates to describe the visual layout of each PDF generated using the service. These templates are customizable but are not starting from a good place. Our job is to modify these templates so they start from our design.

[Finalized PDF template Figma design](https://www.figma.com/file/BiH3LcIVGvzonVmRCk3air/%E2%9D%96-Print-Styles-%7C-DDD?type=design&node-id=140-15612&mode=dev)

## Acceptance Criteria:
1. This is the [repo](https://github.com/GovAlta/adsp-monorepo) you need to work in.
2. The templates are located [here](https://github.com/GovAlta/adsp-monorepo/tree/main/apps/tenant-management-webapp/src/app/store/pdf/defaultTemplates)
3. Set them up to match the design we're given
4. The templates currently feature heavy documentation, which is there to help others use these templates. Maintain this level of documentation when making your changes, this is important.

## Notes
Test that our components can work as is in the templates

## Final Steps
1. Once work is done, submit a PR and let me and @chrisolsen know that it exists.

**Comment by ArakTaiRoth** [comment]
Moving to Blocked status. This is being followed up by the ADSP team and can be checked [here](https://goa-dio.atlassian.net/browse/CS-2590)

**Comment by ArakTaiRoth** [comment]
Follow-up story for this in [design](https://goa-dio.atlassian.net/browse/DDIDS-1255)
